docs/general/polkadot-v1.md

```diff
@@ -86,70 +81,73 @@ Decentralization is a crucial aspect of blockchain networks, but there is a trad
-  every decision goes through public referenda and any token holder can cast a vote.
+- An **on-chain treasury and governance** (see: [OpenGov](../learn/learn-polkadot-opengov.md)) where
+  every decision goes through public referenda, and any token holder can cast a vote.
 
 ### Polkadot's Blockspace
 
-The design and realization of Polkadot 1.0 allowed its creators to enable commoditization of
+The design and realization of Polkadot 1.0 allowed its creators to enable the commoditization of
 blockspace.
 
 A blockchain is a way to store data. The storage unit is the block, and once a block is finalized
 onto the chain, it is practically impossible to modify the data within that block. In addition to
-being tamper-proof, public permissionless blockchains like Polkadot store data that are visible to
-everybody (i.e. public), and anybody can become a network participant permissionlessly.
+being tamper-proof, public permissionless blockchains like Polkadot store data visible to
+everybody (i.e., public), and anybody can become a network participant permissionlessly.
 
 **Blockspace** is the capacity of a blockchain to finalize and commit operations. It represents a
-blockchain's security, computing, and storage capability as an end product. Blockspace produced by
+blockchain's security, computing, and storage capabilities as an end product. Blockspace produced by
 different blockchains can vary in security, flexibility, and availability.
 
-- **Security**, intended as how secure the blockspace is. In Proof-of-Stake (PoS) networks, this is
+- **Security** is intended to show how secure blockspace is. In Proof-of-Stake (PoS) networks, this is
   directly related to how much stake is locked on validator nodes, how much variance in stake there
-  is between validators (i.e. how easy it is to attack a single validator), and how many validators
-  there are securing the network (i.e. how easy it is for colluding validators to exert influence on
+  is between validators (i.e., how easy it is to attack a single validator), and how many validators
+  there are securing the network (i.e., how easy it is for colluding validators to exert influence on
   the network). Additionally, it is also important to look at how many validators are owned by
   single operators (this will determine the degree of social centralization of the network), and how
   many validators run on the same service provider (this will determine the degree of physical
   centralization of the network).
 - **Flexibility**, intended as how flexible the blockspace is, what can be done with it, and what
   type of data can be stored. Data quality plays an important role depending on the type of network.
-  One might avoid having situations in which poor quality data floods blockspace hindering the prompt
+  One might avoid having situations in which poor quality data floods blockspace, hindering the prompt
```

Comment:
This probably is moreso in the realm of security, as security / anti-spam ensures that legit data / blockspace is propgated